collections/Ashmole/MS_Ashmole_1523.xml

```diff
@@ -38,63 +38,635 @@
+                           oculi mei lacrimas per diem et noctem ... (Jeremiah 14: 17–21) (‘In
+                           quadragesima’); </item><item>(11) Recordare domine quid acciderit nobis intuere et
+                           respice obprobrium nostrum ... (Lamentations 5: 1–21) (‘aliud’), ends
+                           imperfectly because of the loss of a leaf after fol. 207; was probably
+                           originally followed by Ezekiel 36: 24–28, now completely missing;</item><item> (12)
+                           Qvis est iste qui uenit de edom tinctis uestibus de bosra ... (Isaiah
+                           63: 1–5), begins imperfectly; </item><item>(13) Uenite et reuertamur ad dominum quia
+                           ipse cepit et sanabit nos ... (Hosea 6: 1–6) (‘aliud’);</item><item> (14) Expecta me
+                           dicit dominus in die resurrectionis mee in futurum ... (Zephaniah 3:
+                           8–13) (‘aliud’); </item><item>(15) Uos sancti domini uocabimini ministri dei nostri .
+                           . . (Isaiah 61: 6–9) (‘De apostolis et pluribus martiribus canticum’);
+                           </item><item>(16) Fvlgebunt iusti et tamquam scintille ... (Wisdom 3: 7–9)
+                           (‘aliud’); </item><item>(17) Reddidit deus mercedem laborum sanctorum suorum ...
+                           (Wisdom 10: 17–20) (‘aliud’);</item><item> (18) Beatus uir qui in sapientia morabitur
+                           ... (Ecclesiasticus 14: 22, 15: 3–4, 6) (‘De uno martire et confessore
+                           cantica’);</item><item> (19) Benedictus uir qui conficit in domino ... (Jeremiah,
+                           17: 7–8) (‘aliud’);</item><item> (20) Beatus uir qui inuentus est sine macula ...
+                           (Sirach 31: 8–11) (‘aliud’); </item><item>(21) Avdite me diuini fructus ... (Sirach
+                           39: 17–21) (‘Unius uirginis . canticum’); </item><item>(22) Gaudens gaudebo in domino
+                           exultabit anima mea ... (Isaiah 61: 10–62: 3) (‘Aliud’);</item><item> (23) Non
+                           uocaberis ultra derelicta ... (Isaiah 62: 4–7) (‘aliud’).</item></list> </note>
+                     </msItem>
+                     <msItem n="8">
+                        <locus>(fols. 212v–258r)</locus><title type="desc"> Cluniac hymnal</title><note> With short rubrics and musical
+                           notation for the opening lines of hymns. Includes hymns for the week with
+                           variations during the year, hymns for feasts and saints’ days including
+                           Agatha, Benedict, Pancras, John the Baptist, Peter and Paul, Mary
+                           Magdalene, Michael and Martin. Fol. 258v is ruled but blank.</note>
+                        <msItem>
+                           <incipit>O lux beata trinitas</incipit><note>(Chevalier, no. 13150)</note>
```

Comment:
These are also in Cantus, e.g. this one is <http://cantusindex.org/id/008358>, but I am unsure whether it's worth the effort to add these references.